brainwallet/Bento Effects /ThemeGradientHelpers.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ThemeGradientHelpers.swift
+//  brainwallet
+//
+//  Created by Kerry Washington on 17/10/2025.
+//  Copyright © 2025 Grunt Software, LTD. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+ enum MainGradientStyle {
+    case darkStyle
+    case lightStyle
+
+    var colorThemeArray: [Color] {
+        switch self {
+        case .darkStyle:
+            return  [BrainwalletColor.surface.opacity(0.2), BrainwalletColor.grape.opacity(0.2), BrainwalletColor.midnight.opacity(0.1)]
+        case .lightStyle:
+            return [BrainwalletColor.surface.opacity(0.1), BrainwalletColor.grape.opacity(0.1), BrainwalletColor.midnight.opacity(0.1)]
+        }
+    }
+
+     var maskGradientStops: [Color] {
+         switch self {
+         case .darkStyle:
+             return [.black,.black,.black,.black,.clear]
+         case .lightStyle:
+             return [.black,.black,.clear,.clear,.clear]
+         }
+     }
+ }
+
+struct BentoShadow: ViewModifier {
+
+    @Binding
+    var userPrefersDarkTheme: Bool
+    func body(content: Content) -> some View {
+        content
+            .shadow(color: userPrefersDarkTheme ? Color.black.opacity(0.8) :
+                        Color(red: 0.341176, green: 0.329411, blue: 1).opacity(0.12),
+                    radius: 2, x: 1, y: 2.4)
+        /// Dark Mode : #000000 at 40% Opacity
```

Comment:
Pulled the values from the Figma @JosephSanjaya . Again @trk15  has a big say on how the final look is. But we should be very close to matching for the shadows